src/components/ReferredRecord/ReferredRecord.test.js

```diff
@@ -0,0 +1,35 @@
+import '../../../test/jest/__mock__';
+import { render, screen } from '@testing-library/react';
+import ReferredRecord from './ReferredRecord';
+
+const propsData = {
+  instanceId: 'testInstanceId',
+  instanceTitle: 'testInstanceTitle',
+  itemBarcode: '90909090',
+  itemId: 'testItemId',
+  holdingsRecordId: 'testHoldingsRecordId',
+  requestCreateDate: '2023-04-05',
+  requesterId: 'testRequesterId',
+  requesterName: 'testRequesterName',
+};
+
+const renderReferredRecord = (prop) => render(<ReferredRecord values={prop} />);
+
+describe('ReferredRecord', () => {
+  it('entityType.request should render', () => {
+    renderReferredRecord(propsData);
+    expect(screen.getByText('ui-requests.notes.entityType.request')).toBeInTheDocument();
+  });
+  it('assigned.for should render', () => {
+    renderReferredRecord(propsData);
+    expect(screen.getByText('ui-requests.notes.assigned.for')).toBeInTheDocument();
+  });
+  it('assigned.requester should render', () => {
+    renderReferredRecord(propsData);
+    expect(screen.getByText('ui-requests.notes.assigned.requester')).toBeInTheDocument();
+  });
+  it('assigned.requestDate should render', () => {
+    renderReferredRecord(propsData);
+    expect(screen.getByText('ui-requests.notes.assigned.requestDate')).toBeInTheDocument();
+  });
```

Comment:
Could you use `beforeEach` to trigger rendering of component and get rid of rendering inside each test?

Also would be great if you can move all labels like `'ui-requests.notes.assigned.requester'` to separate object `labelIds={}` and then use that object? (see example [here](https://github.com/folio-org/ui-requests/blob/6c369ffea263614f788a0dbe789f48434daf60d7/src/components/ErrorModal/ErrorModal.test.js#L18))